docs/vc-http-api.yml

```diff
@@ -179,12 +239,18 @@ components:
         checks:
           type: array
           description: The checks performed
+          items:
+            type: string
         warnings:
           type: array
           description: Warnings
+          items:
+            type: string
         errors:
           type: array
           description: Errors
+          items:
+            type: string
       example: { "checks": ["proof"], "warnings": [], "errors": [] }
```

Comment:
I continue to hate this

Reply:
Related: https://github.com/w3c-ccg/vc-http-api/issues/74. Each check should be able to express its own error. I can maybe try to proposa a separate PR for that.

Reply:
Agree... `items` and `data` are the ultimate "I gave up trying to pick a good name". :P

Reply:
Understood however I was just fixing a syntax error in this section and would prefer not to expand the scope of this PR to addressing these issues, lets fix with #74